node/testdata/test.ts

```diff
@@ -0,0 +1,40 @@
+import { walk } from "../../fs/walk.ts";
+import { basename, dirname, fromFileUrl } from "../../path/mod.ts";
+import { assertEquals, fail } from "../../testing/asserts.ts";
+import { config, testList } from "./common.ts";
+
+/**
+ * This script will run the test files specified in the configuration file
+ * 
+ * Each test file will be run independently and wait until completion, if an abnormal
+ * code for the test is reported, the test suite will fail inmediately
+ */
+
+const dir = walk(fromFileUrl(new URL(config.suitesFolder, import.meta.url)), {
+  includeDirs: false,
+  match: testList,
+});
+
+for await (const file of dir) {
+  Deno.test({
+    name: `Node test runner: ${basename(file.path)}`,
```

Comment:
Last nit; maybe just let this be the whole path, or at-least the relative path from the node dir so that it's obvious (once we have a lot of tests) which test suite it belongs to.

Reply:
Done